tests/main.test.ts

```diff
@@ -47,22 +47,66 @@ describe("src/main.ts", function() {
 
     const validAny = validModel(anyType);
 
-    it('should allow all types when no restriction', function(done) {
+    it('should allow all types for -any- schema when there are no restriction',
```

Comment:
NIT: it('should allow all types when validating against an anySchema with no restrictions',